cloudify_async_client/client.py

```diff
@@ -27,9 +27,19 @@ def url(self):
 
     def get(self, url, **kwargs):
         timeout = kwargs.pop('timeout', 300)
+        if timeout is None:
+            timeout = 300
         if isinstance(timeout, int) or isinstance(timeout, float):
             timeout = aiohttp.ClientTimeout(total=timeout)
+
+        # Format query parameters
+        params = kwargs.copy()
+        p = {k: str(v) for k, v in params.pop('params', {}).items()
+             if v is not None}
```

Comment:
this kinda sucks, we use `params` too many times. I suggest `def get(self, url, params=None, **kwargs):`, and then `if params: params={k:str(v) for k, v}`, and then `self.session.get(..., params=params, **kwargs)`

Reply:
which, actually _do_ we need the str()?

Reply:
we do, because somewhere in the depths of aiohttp's ClientSession there is type-checking on the parameters' values